pkg/kubelet/cm/container_manager_linux.go

```diff
@@ -204,7 +215,15 @@ func NewContainerManager(mountUtil mount.Interface, cadvisorInterface cadvisor.I
 			"This will be a fatal error by default starting in K8s v1.6! " +
 			"In the meantime, you can opt-in to making this a fatal error by enabling --experimental-fail-swap-on.")
 	}
+	var capacity = v1.ResourceList{}
+	if info, err := cadvisorInterface.MachineInfo(); err == nil {
+		capacity = cadvisor.CapacityFromMachineInfo(info)
+	} else {
+		return nil, err
+	}
 
+	cgroupRoot := nodeConfig.CgroupRoot
```

Comment:
fyi @sjenning -- can you take the todo to make sure we have a call that basically dumps the output of node config into a log so we can handle a variety of bad support cases in the future with sane logs?